src/state/actions/search.js

```diff
@@ -0,0 +1,67 @@
+import fetch from 'node-fetch';
+import ActionTypes from './action-types';
+
+/**
+ * requestSearch - action creator
+ *
+ * @param  {String} targetId
```

Comment:
what is targetId here?

Reply:
`targetId` is likely just the `manifestId` most of the time. If we extend it further it could be other things I think